.github/workflows/deploy-apis-to-production.yml

```diff
@@ -1750,8 +1741,9 @@ jobs:
         with:
           action: fjogeleit/yaml-update-action@main # Action to retry
           with: |
-            valueFile: "k8s/network-uptime/values-prod.yaml"
-            propertyPath: "image.tag"
+            valueFile: "k8s/predict/new_directory/values-prod.yaml"
+            
+            propertyPath: "image.new_property"
```

Comment:
Hi @petermark23 
What is the `new_property` value?
Could you be meaning `tag`?